### PR TITLE
Initial Implementation of loss database and SWIO loss data import

### PR DESCRIPTION
**Pull request by pslh**
Loss Database schema, import scripts to read JSON meta-data files
JSON meta-data files for SWIO Average Annual Losses and Return Period loss curves
Support scripts to generate SQL and JSON for loss value import

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+## Quick setup
+
+### Pre-requisites
+
+Ubuntu
+```bash
+$ sudo apt install python-virtualenv python-dev
+```
+CentOS/FEdora
```
Comment (daniviga) [inline README.md]: `s/FEdora/Fedora/`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+```bash
+$ sudo yum install python-virtualenv python-devel
+```
+### Installation
+
+```bash
+$ virtualenv gedenv
+$ source gedenv/bin/activate
+$ pip -U install pip
+$ pip install -r requirements.txt
```
Comment (daniviga) [inline README.md:25]: Is Python 2 still the reference?

File: data/swio/import-loss-shp.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+then
+	echo "Usage $0 <shapefile>"
+	exit 1
+fi
+
+filename=$1
+table_name=$(basename $filename .shp)
+
+echo "shp2pgsql -c -W UTF-8 -s 4326 $filename temp.$table_name  > $table_name.sql"
+shp2pgsql -c -W UTF-8 -s 4326 $filename temp.$table_name  > $table_name.sql
```
Comment (daniviga) [inline data/swio/import-loss-shp.sh]: The script will not work if the shp has spaces in the name.

Either we consider this ok or you need to put `$filename` inside double quotes

**Review by daniviga — APPROVED** [review]
LGTM

File: data/swio/gen-json.sh [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+sed -e 's/@COUNTRY@/Seychelles/g' -e 's/@ISO@/syc/g' -e 's/@PERIL@/Earthquake/g' -e 's/@TH@/eq/g' template.json > syc-eq.json
+sed -e 's/@COUNTRY@/Seychelles/g' -e 's/@ISO@/syc/g' -e 's/@PERIL@/Tropical Cyclone/g' -e 's/@TH@/tc/g' template.json > syc-tc.json
+sed -e 's/@COUNTRY@/Seychelles/g' -e 's/@ISO@/syc/g' -e 's/@PERIL@/Non-Tropical Cyclone/g' -e 's/@TH@/ntc/g' template.json > syc-ntc.json
+sed -e 's/@COUNTRY@/Seychelles/g' -e 's/@ISO@/syc/g' -e 's/@PERIL@/All Perils/g' -e 's/@TH@/ap/g' template.json > syc-ap.json
+
+sed -e 's/@COUNTRY@/Zanzibar/g' -e 's/@ISO@/zan/g' -e 's/@PERIL@/Earthquake/g' -e 's/@TH@/eq/g' template.json > zan-eq.json
+sed -e 's/@COUNTRY@/Zanzibar/g' -e 's/@ISO@/zan/g' -e 's/@PERIL@/Tropical Cyclone/g' -e 's/@TH@/tc/g' template.json > zan-tc.json
+sed -e 's/@COUNTRY@/Zanzibar/g' -e 's/@ISO@/zan/g' -e 's/@PERIL@/Non-Tropical Cyclone/g' -e 's/@TH@/ntc/g' template.json > zan-ntc.json
+sed -e 's/@COUNTRY@/Zanzibar/g' -e 's/@ISO@/zan/g' -e 's/@PERIL@/All Perils/g' -e 's/@TH@/ap/g' template.json > zan-ap.json
```
Comment (micheles) [inline data/swio/gen-json.sh:24]: Matteo would be proud of you

Comment (daniviga) [reply data/swio/gen-json.sh:24]: +1

File: python/import_loss_model.py [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+import json
+import sys
+
+from loss_model import LossModel
+# , LossMap, LossCurveMap
+from cf_common import Contribution, License
+
+from django.db import connections
+from django.conf import settings
+
```
Comment (micheles) [inline python/import_loss_model.py:34]: Out of curiosity, why are using Django when there are no Django models here? The idea is to add a Django-based WebUI later on?

Comment (daniviga) [reply python/import_loss_model.py:34]: I guess just to use its DB connections abstraction layer (_not_ the ORM)

Comment (pslh) [reply python/import_loss_model.py:34]: Essentially just for connections, cursors and DB settings.  I do not want an ORM or a web UI at this point.

File: README.md (hunk 2 above)
Comment (pslh) [reply README.md:25]: Yes, for now.

**Pull request merged**